### PR TITLE
docs: update README recording section

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR updates the [README.md: record-test-results-on-cypress-cloud](https://github.com/cypress-io/github-action/blob/master/README.md#record-test-results-on-cypress-cloud) section.

- The syntax for single event triggers in the text of this section is simplified (see also https://github.com/cypress-io/github-action/pull/782). `on: [push]` changes to `on: push`
- The config file is updated from the legacy name `cypress.json` to the refer to the documentation [Cypress Configuration File](https://docs.cypress.io/guides/references/configuration#Configuration-File) instead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -342,13 +342,13 @@ jobs:
...
 
 **Tip 2:** we recommend passing the `GITHUB_TOKEN` secret (created by the GH Action automatically) as an environment variable. This will allow correctly identifying every build and avoid confusion when re-running a build.
 
 **Tip 3:** if running on `pull_request` event, the commit message is "merge SHA into SHA", which is not what you want probably. You can overwrite the commit message sent to Cypress Cloud by setting an environment variable. See [issue 124](https://github.com/cypress-io/github-action/issues/124#issuecomment-653180260) for details.
 
-**Tip 4:** to record the project needs `projectId`. Typically this value is saved in the `cypress.json` file. If you want to avoid this, pass the project id using an environment variable:
+**Tip 4:** to record the project needs `projectId`. Typically this value is saved in the [Cypress Configuration File `cypress.json.*`](https://docs.cypress.io/guides/references/configuration#Configuration-File). If you want to avoid this, pass the `projectId` using an environment variable:
```
[the remark this answers is not in the document]
Comment (MikeMcC399) [reply README.md:351]: That is fine with me.

File: README.md [hunk 2]
```diff
@@ -342,13 +342,13 @@ jobs:
 
 [![recording example](https://github.com/cypress-io/github-action/workflows/example-recording/badge.svg?branch=master)](.github/workflows/example-recording.yml)
 
-**Tip 1:** We recommend using the action with `on: [push]` instead of `on: [pull_request]` to get the most accurate information related to the commit on the dashboard. With pull requests, the merge commit is created automatically and might not correspond to a meaningful commit in the repository.
```
[the remark this answers is not in the document]
Comment (MikeMcC399) [reply README.md:345]: `[...]` does work, it is the syntax for 

- [Using multiple events](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-multiple-events)

A while back I simplified all the other single event triggers in PR https://github.com/cypress-io/github-action/pull/782 and took out the brackets. I missed these two cases however and I wanted to make the README consistent, using only the syntax described in

- [Using a single event](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-a-single-event)

where there was only a single event set up as a trigger.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MikeMcC399) [reply README.md]: Done in https://github.com/cypress-io/github-action/pull/874/commits/4249b007aa0cce7e314246ff68678505695a660c

**Comment by MikeMcC399** [comment]
Current changed section is on

[README: Record test results on Cypress Cloud](https://github.com/MikeMcC399/github-action/blob/docs/recording/README.md#record-test-results-on-cypress-cloud) 

for review.

**Review by jordanpowell88 — APPROVED** [review]

**Review by marktnoonan — APPROVED** [review]

**Pull request merged**